assets/js/blocks/classic-template/constants.ts

```diff
@@ -69,4 +75,33 @@ export const TEMPLATES: TemplateDetails = {
 		),
 		placeholder: PLACEHOLDERS.archiveProduct,
 	},
+	cart: {
+		type: TYPES.cart,
+		title: __( 'WooCommerce Cart Block', 'woo-gutenberg-products-block' ),
```

Comment:
Do you think the WooCommerce Prefix here can be removed to make the titles more succinct? It should be fairly obvious these are WooCommerce blocks from the context and icons.

Reply:
I agree, but since others also use the WooCommerce prefix, perhaps this should be changed as a whole after merging templates. Also not sure where this is actually used 🤔

Reply:
This is shown when you focus on the legacy block in the editor.